stl/inc/__msvc_sanitizer_annotate_container.hpp

```diff
@@ -121,9 +156,7 @@ void __cdecl __sanitizer_annotate_contiguous_container(
 #error Unknown architecture
 #endif // ^^^ unknown architecture ^^^
 
-#endif // insert asan annotations
-
-#endif // !defined(_M_CEE_PURE) && asan not disabled
+#endif // ^^^ insert asan annotations

```

Comment:
Not for this PR, but we really should consistently capitalize "ASan" in docs and comments to avoid confusion. (No change requested.)